package.json

```diff
@@ -347,6 +355,11 @@
 				}
 			],
 			"view/item/context": [
+				{
+					"command": "cosmosDB.selectSubscriptions",
```

Comment:
Remember to exclude this from the command palette

Reply:
done